app/scripts/lib/mailcheck.js

```diff
@@ -55,37 +55,34 @@ define(function (require, exports, module) {
 
           // user got a suggestion to check their email input
           view.notifier.trigger('mailcheck.suggested');
-          if (view.isInExperimentGroup('mailcheck', 'treatment')) {
+          const message = view.unsafeTranslate(DID_YOU_MEAN_TEXT, {
+            escapedDomain: _.escape(suggestion.domain)
+          });
 
-            const message = view.unsafeTranslate(DID_YOU_MEAN_TEXT, {
-              escapedDomain: _.escape(suggestion.domain)
-            });
+          let tooltip = new Tooltip({
+            dismissible: true,
+            extraClassNames: SUGGEST_DIV_CLASS,
+            invalidEl: target,
+            type: 'mailcheck',
+            unsafeMessage: message
+          });
 
-            let tooltip = new Tooltip({
-              dismissible: true,
-              extraClassNames: SUGGEST_DIV_CLASS,
-              invalidEl: target,
-              type: 'mailcheck',
-              unsafeMessage: message
-            });
+          tooltip.render();
+          // set that this value was suggested, user might not click on the tooltip
+          // but still take the suggestion
+          element.data('mailcheckValue', suggestion.full);
 
-            tooltip.render();
-            // set that this value was suggested, user might not click on the tooltip
-            // but still take the suggestion
-            element.data('mailcheckValue', suggestion.full);
-
-            tooltip.$el.on('click keypress', 'span', function (e) {
-              // if a click event is triggered or an enter key is pressed, destroy
-              // the tooltip.
-              if (e.type === 'click' || e.which === KeyCodes.ENTER) {
-                element.val(suggestion.full);
-                // the user has used the suggestion
-                view.notifier.trigger('mailcheck.clicked');
```

Comment:
@divyabiyani this used to log that the tooltip was clicked into the experiment file. Could you change this to `logEvent`, this way we still get metrics that users click on the tooltip. 

We might have a unit test for the mailcheck click action, we need to update it to make sure that the correct event is logged